### DISCUSSION TITLE
Apply Connaisseur to cluster without helm

### DISCUSSION THREAD

**Discussion by lacuchilla** [sse-secure-systems/connaisseur#808] [Q&A] [answered]
Is there a recommended method of applying Connaisseur to a Kubernetes cluster without using helm?

**Comment by phbelitz** [comment, marked as answer]
No, not at the moment. The usage of Helm is rather critical, since it allows us to make the deployment in a specific order. Some of the components of Connaisseur MUST be deployed before others, otherwise the whole deployment will break apart ...
